### ISSUE TITLE
README:setup, doc

### ISSUE THREAD

**Issue by simonjwright** [AdaCore/langkit#430]
README.md doesn’t mention the need to call `setup.py`, or how to call it (the documentation suggests using `setup.py/easy_install/pip/...`, whatever `...` is, but we’ve already installed `REQUIREMENTS.dev`).

Also, it says the documentation is found in `langkit/doc`, but it’s actually in `doc`.

People might get started quicker if `eval $(./manage.py setenv)` was mentioned?

**Comment by pmderodat** [comment]
Hello Simon,

Thank you for pointing that out. We have been reorganizing the Langkit project for several months, now. It’s not over yet, but we’ll try to update the readme to be useful ASAP in any case. :-)

Just being curious: did you notice this by chance, or are you trying to package Langkit in some distribution?

**Comment by simonjwright** [comment]
> Just being curious: did you notice this by chance, or are you trying to package Langkit in some distribution?

I’ve been trying to get GNATstudio running with (a) Python3, (b) macOS Catalina. I’d picked up that libadalang depends on langkit, but what I hadn’t realised was that libadalang would manage this dependency itself, so I started off building langkit, hence the problems.

Another thing I noticed: running langkit’s `setup.py install` installed the langkit python module in such a way that libadalang’s REQUIREMENTS.dev failed (couldn’t overwrite langkit, so rest of requirements were omitted).

Just tried building & installing libadalang in an older machine without any previous installs of libadalang or langkit; worked just fine.
